content/en/docs/contribute/participate/issue-wrangler.md

```diff
@@ -0,0 +1,58 @@
+---
+title: Issue Wranglers
+content_type: concept
+weight: 20
+---
+
+<!-- overview -->
+
+In order to reduce the burden on the [PR Wrangler](/docs/contribute/participate/pr-wrangler), formal approvers, and reviewers, members of SIG Docs take week long shifts [triaging and categorising issues](/docs/contribute/review/for-approvers.md/#triage-and-categorize-issues) for the repository.
+
+<!-- body -->
+
+## Duties
+
+Each day in a week-long shift as Issue Wrangler:
+
+- Triage and tag incoming issues daily. See [Triage and categorize issues](https://github.com/kubernetes/website/blob/main/docs/contribute/review/for-approvers/#triage-and-categorize-issues) for guidelines on how SIG Docs uses metadata.
+- Identifying whether the issue falls under the support category and assigning a "triage/accepted" status.
+- Assuring the issue is tagged with the appropriate sig/area/kind labels.
+- Keeping an eye on stale & rotten issues within the kubernetes/website repository.
+- [Issues board](https://github.com/orgs/kubernetes/projects/72/views/1) maintenance would be nice
+
+### Requirements
+
+- Must be an active member of the Kubernetes organization.
+- A minimum of 15 quality contributions to Kubernetes (of which a certain amount should be directed towards kubernetes/website).
+- Performing the role in an informal capacity already
+
+### Helpful Prow commands for wranglers
+
+```
+# reopen an issue
+/reopen
+
+# transfer issues that don't fit in k/website to another repository
+/transfer[-issue]
+
+# change the state of rotten issues
+/remove-lifecycle rotten
+```
+
```

Comment:
Can also add other commands such as:
 
- /remove-lifecycle stale 
- /sig <sig_name> => This is used to assign the sig to an issue
- /area <area_name> => Used to add a specific area (look up the k8s-ci-robot commands list for areas you can add
- //kind support => Tagging issue as support-specific
- /help wanted
- /good-first-issue
- /triage accepted => to accept triaging for an issue

Reply:
Would a link to the prow commands be helpful?
https://prow.k8s.io/command-help